packages/client/hmi-client/src/components/workflow/scenario-templates/scenario-template-utils.ts

```diff
@@ -0,0 +1,34 @@
+import { DistributionType } from '@/services/distribution';
+import { ParameterSemantic } from '@/types/Types';
+import { calculateUncertaintyRange } from '@/utils/math';
+
+export const displayParameter = (parameters: ParameterSemantic[], parameterName: string) => {
+	let value = '';
+	const parameter = parameters.find((p) => p.referenceId === parameterName);
+	switch (parameter?.distribution.type) {
+		case DistributionType.Constant:
+			value = `${parameter.distribution.parameters.value}`;
+			break;
+		case DistributionType.Uniform:
+			value = `${parameter.distribution.parameters.minimum} - ${parameter.distribution.parameters.maximum}`;
+			break;
+		default:
+			return '';
+	}
+	return `${parameterName}  [${value}]`;
+};
+
+export const formatUniformDistribution = (parameter: ParameterSemantic) => {
```

Comment:
The function name is somewhat misleading here. You are not formatting a uniform-distribution, you are coercing non-uniform into a uniform format. A better choice of naming would be:
- convertToUniformDistrivution
- switchToUniformDistribution 
- ...etc

Reply:
Gotcha, I'm now using `switchToUniformDistribution`